README.md

```diff
@@ -54,30 +129,60 @@ const credential = await navigator.credentials.get({
 });
 ```
 
-## Confirming payment
+## Authenticating a payment
 
-Any origin may invoke the [Payment Request API](https://w3c.github.io/payment-request/) to prompt the user to verify a credential created by any other origin. The `PaymentRequest.show()` method must require a user gesture and display user interface with the amount of payment and the hostname of the top-level context where the `PaymentRequest` API was invoked.
+Any origin may invoke the [Payment Request API](https://w3c.github.io/payment-request/) with the `secure-payment-confirmation` payment method to prompt the user to verify a SecurePaymentCredential created by any other origin. The `PaymentRequest.show()` method must require a user gesture. The browser will display a native user interface with the payment amount and the payee origin, which is taken to be the origin of the top-level context where the `PaymentRequest` API was invoked.
```

Comment:
> The `PaymentRequest.show()` method must require a user gesture.

That's a given, as it's baked into Payment Request... maybe just a statement of fact: "The `PaymentRequest.show()` method requires a user gesture."